test/system/252-quadlet.bats

```diff
@@ -233,13 +266,13 @@ EOF
     service_cleanup $QUADLET_SERVICE_NAME inactive
 }
 
-@test "quadlet - ContainerName" {
+@test "quadlet - ContainerName and journal output check" {
     local quadlet_file=$PODMAN_TMPDIR/containername_$(random_string).container
     cat > $quadlet_file <<EOF
 [Container]
 ContainerName=customcontainername
 Image=$IMAGE
-Exec=top"
+Exec=sh -c "echo I AM AN OUTPUT; echo I AM AN ERROR 1>&2; top -d 10"
```

Comment:
Suggestion: define `token_out="stdout$(random_string 10)"` and `token_err="stderr$(random_string 10)"` above, then use them in the echo and test.